Floofbot/Services/RaidProtectionService.cs

```diff
@@ -372,7 +431,7 @@ public async Task<bool> CheckMessage(FloofDataContext _floofDb, SocketMessage ms
 
             if (spammedMentions)
                 return false; // user already banned
-            if (userSpammedMessages || userSpammedLetters || userSpammedInviteLink || userSpammedEmojis)
+            if (filteredWord || userSpammedMessages || userSpammedLetters || userSpammedInviteLink || userSpammedEmojis)
```

Comment:
This got me thinking, what if warnings compounded on each other? For example, if a user spammed a filtered word _and_ messages, should that count double?

Also there could be a concept of infraction weights. Spamming emojis is a less severe action than triggering the word filter.

Reply:
I think these can be added as an issue and fixed in a future PR